Fix broken hyperlink

droctothorpe opened the pull request:
#### What type of PR is this?
/kind documentation

#### What this PR does / why we need it:
This tiny PR just fixes a broken hyperlink in the FAQ.

#### Does this PR introduce a user-facing change?
<!--
If no, just write "NONE" in the release-note block below.
If yes, a release note is required:
Enter your extended release note in the block below. If the PR requires additional action from users switching to the new release, include the string "action required".

For more information on release notes see: https://git.k8s.io/community/contributors/guide/release-notes.md
-->
```release-note
None
```

Shubham82 commented on cluster-autoscaler/README.md:
```diff
@@ -10,7 +10,7 @@ Cluster Autoscaler is a tool that automatically adjusts the size of the Kubernet
 # FAQ/Documentation
 
-An FAQ is available [HERE](./FAQ.md).
+A FAQ is available [HERE](./FAQ.md).
```
IMO, `An FAQ`  is grammatically accurate.

lzhecheng replied:
Agree

droctothorpe replied:
Duly noted. Removed from the PR. Thanks!

Shubham82 commented on cluster-autoscaler/FAQ.md:
```diff
@@ -862,7 +862,7 @@ Events:
 
 This limitation was solved with
-[volume topological scheduling](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/storage/volume-topology-scheduling.md)
+[volume topological scheduling](https://github.com/kubernetes/design-proposals-archive/blob/acc25e14ca83dfda4f66d8cb1f1b491f26e78ffe/storage/volume-topology-scheduling.md)
```
```suggestion
[volume topological scheduling](https://github.com/kubernetes/design-proposals-archive/blob/main/storage/volume-topology-scheduling.md)
```

Shubham82 replied:
Please change the branch name to main.

droctothorpe commented on cluster-autoscaler/README.md:
```diff
@@ -10,7 +10,7 @@ Cluster Autoscaler is a tool that automatically adjusts the size of the Kubernet
 # FAQ/Documentation
 
-An FAQ is available [HERE](./FAQ.md).
+A FAQ is available [HERE](./FAQ.md).
```
```suggestion
An FAQ is available [HERE](./FAQ.md).
```

droctothorpe commented:
Thanks for the input. Merged suggestions. Squashed. Kept you as co-author, @Shubham82. Updated commit message, PR title, and PR description accordingly.

What's the joke?
1 line PR: 10 comments.
10,000 line PR: 0 comments. 

🙃

Shubham82 commented:
Thanks, @droctothorpe 
it looks good to me.
/lgtm

droctothorpe commented:
What's the next step, @Shubham82?

Shubham82 commented:
@droctothorpe you can ask [feiskyer](https://github.com/feiskyer) for approval by assiging the issue to him.
See this [comment](https://github.com/kubernetes/autoscaler/pull/5987#issuecomment-1652249884)

mwielgus reviewed (approved):
/lgtm
/approve

droctothorpe commented:
Thank you, frondz! Does this mean I get to go to the contributor lounge at kubecon? 😂 Hopefully more substantive contributions to follow.
